src/lib/supabase.ts

```diff
@@ -1,32 +1,10 @@
-import { createClient } from '@supabase/supabase-js';
+import {createClient, User} from '@supabase/supabase-js';
 
-// probably should move these to an env.
-export const supabase = createClient(import.meta.env.PUBLIC_SUPABASE_URL,
-  import.meta.env.PUBLIC_SUPABASE_API_KEY);
+export const supabase = createClient(
+  import.meta.env.VITE_SUPABASE_URL,
+  import.meta.env.VITE_SUPABASE_API_KEY);
```

Comment:
Should make this a 2.0.0 change with this. Is it possible to implement a setup script to copy a gist for the env setup? Just thinking of thoughts on how to mitigate this breaking change. 

I suppose we can share a note in the discord too.

Reply:
Gist is my least favorite method as leaking that we wouldn't be able to tell what happened. New variables will be locked in for local development, I just copy-pasted the wrong `.gitignore` :D. Just add the new variables and all should work, delete the old ones once this is merged

Reply:
reverted the BC on this, we have the same limitations as open-sauced and previous hot instance :D